contracts/HydraStaking/HydraStaking.sol

```diff
@@ -95,6 +95,20 @@ contract HydraStaking is
         distributedRewardPerEpoch[epochId] = totalReward;
     }
 
+    /**
+     * @inheritdoc IHydraStaking
+     */
+    function temporaryRemove(address account) external onlyHydraChain {
+        emit BalanceChanged(account, 0);
```

Comment:
On the finish ban, we also sync (using stake + delegated), while here we set 0 for both stake and delegate.
Could that cause problems?

Reply:
the idea is to signal the nodes that this validator must be removed from the validator set which happens when setting the balance to zero.